helpers.lua

```diff
@@ -12,7 +12,10 @@ local pairs = pairs
 local rawget = rawget
 local require = require
 local tonumber = tonumber
-local io = { open = io.open }
+local io = { 
+    open = io.open, 
+    popen = io.popen 
+}
```

Comment:
here `uname` is used to distinguish the operating system once. This information should be also exposed to the user in case configuration should work on multiple platforms. We also need a helper, which set a platform specific function at the end of the module ```return setmetatable(fs, { __call = function(_, ...) return worker(...) end })``
Platform specific code should be in dedicated files suffixed by the platform for example `_linux`, `_freebsd`,
the helper gets the supported platforms as arguments and will either load such files or return an error (platform not supported).